SNOW-1279647: Update chairlift quickstart steps to use SnowCLI (#1157)
Snowflake-Labs/sfquickstarts: site/sfguides/src/native-app-chairlift/native-app-chairlift.md

```

<!-- ------------------------ -->

> ## Create application package and install application using SnowCLI

Reviewer: ```suggestion
## Create application package and install application using Snowflake CLI
```

For more details, see the comments in the **provider/create-package.sql** script.

**Deploy the application package**

Open a new terminal in the root of the repository and execute the following command:

```bash
snow app run
```

> This command will upload source files and create the application package and install the application instance automatically. 

Reviewer: ```suggestion
This command will upload source files, create the application package, and install the application object automatically. When you run it again, it will perform the minimum steps necessary to ensure the application is up-to-date with your local copy. 
```

To do this, execute the following SQL statement using role **chairlift_provider**:

```sql
use role chairlift_provider;
grant install, develop on application package chairlift_pkg to role chairlift_admin;
```bash
snow app run
```

Next, execute the **consumer/install-app.sql** script. This script installs the application in the account, and grants appropriate privileges to the **chairlift_viewer** role. Note that the version and/or patch values may need to be updated to install the application using a different version or patch.
This command will upload source files and create the application package and install the application instance automatically. 

```sql
use role chairlift_admin;
use warehouse chairlift_wh;
Snowflake CLI project is configured using `snowflake.yml` file.

-- create the actual application from the package in versioned dev mode
create application chairlift_app
    from application package chairlift_pkg
    using version develop;

-- allow our secondary viewer role restricted access to the app
grant application role chairlift_app.app_viewer

>     to role chairlift_viewer;

Reviewer: Where is this being done in the new flow?

## Clean up
Duration: 1

> Let's clean up your Snowflake account. In the same terminal you open before, execute the following command:

Reviewer: ```suggestion
Let's clean up your Snowflake account. In the same terminal you opened before, execute the following command:
```

## Clean up
Duration: 1

Let's clean up your Snowflake account. Run the following SQL commands in a worksheet:
Let's clean up your Snowflake account. In the same terminal you opened before, execute the following command:

```sql
----- CLEAN UP -----
USE ROLE chairlift_admin;
DROP APPLICATION chairlift_app;
DROP DATABASE chairlift_consumer_data;

USE ROLE chairlift_provider;
DROP APPLICATION PACKAGE chairlift_pkg;

USE ROLE ACCOUNTADMIN;
DROP WAREHOUSE chairlift_wh;
DROP ROLE chairlift_provider;
DROP ROLE chairlift_viewer;

> DROP ROLE chairlift_admin;

Reviewer: @sfc-gh-osalazarlizano can you re-add all the other lines other than `drop application` and `use role chairlift_provider; drop application package`? We still need to clean up those other things as they won't be dropped by `snow app teardown`.
